presto-native-execution/presto_cpp/main/PeriodicTaskManager.h

```diff
@@ -146,6 +168,10 @@ class PeriodicTaskManager {
   // Renabled this after update velox.
   velox::common::SpillStats lastSpillStats_;
   velox::memory::MemoryArbitrator::Stats lastArbitratorStats_;
+
+  // CAUTION: Declare last since the threads access other members of `this`.
+  folly::FunctionScheduler onceRunner_;
+  folly::ThreadedRepeatingFunctionRunner repeatedRunner_;
```

Comment:
What's the difference ThreadedRepeatingFunctionRunner than FunctionScheduler? Thanks!